### DISCUSSION TITLE
How to Update SSID and Password using MQTT Client?

### DISCUSSION THREAD

**Discussion by KarthikHanasiEvbooth** [arendst/Tasmota#15797] [Q&A] [answered]
Hello, I am trying to update SSID and Password of **Sonoff POWR2** configured with Tasmota, using MQTT client (MQTT.fx) and MQTT. But unable to update SSID and Password of the Device from MQTT.

Command using to update SSID and Password is 

**_mqttClient.sendMessage('cmnd/topic_123/RESULT', 'SSID2 newUSername; Password2 123456');_**

above one doing it from Node JS MQTT.

Please help me to Update SSID and Password.

Thank you.

**Comment by barbudor** [comment, marked as answer]
The topic for a given command `Xxxxx` is `cmnd/topic_123/Xxxx` 
So to change the SSID2, it should be
Topic: `cmnd/topic_123/SSID2`
Payload : the SSID name

However as Tasmota may disconnect and reboot after the SSID command you should use a Backlog command to send both SSID and Password in a single shot using:
Topic: `cmnd/topic_123/Backlog`
Payload: `SSID2 newssid; Password2 newpassword`

**Reply by KarthikHanasiEvbooth** [reply]
Yes **Thank you so much**, problem was solved !!!
